services/static-webserver/client/source/class/osparc/dashboard/ResourceMoreOptions.js

```diff
@@ -40,45 +42,23 @@ qx.Class.define("osparc.dashboard.ResourceMoreOptions", {
   },
 
   statics: {
-    WIDTH: 820,
-    HEIGHT: 720,
+    WIDTH: 1035,
+    HEIGHT: 700,
 
     popUpInWindow: function(moreOpts) {
-      const title = qx.locale.Manager.tr("Details");
+      // eslint-disable-next-line no-underscore-dangle
+      const title = qx.locale.Manager.tr(`Project Details - ${moreOpts.__resourceData.name}`);
```

Comment:
Do not use "Project", use ``osparc.product.Utils.getStudyAlias({firstUpperCase: true`` instead